From: Martin Kletzander <mkletzan@redhat.com>
Subject: [RHEL-6.4 python-virtinst PATCH] Update pod file from latest pod.in
Date: Wed, 19 Dec 2012 17:56:44 +0100
Message-Id: <88d2285e89778a405e6bae8b5d7c42db8ce7ae61.1355932223.git.mkletzan@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=832339

Updating to latest version of pod.in is the last thing needed in order
to make the bug go away.

RHEL-only
---
 man/en/virt-install.pod | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/man/en/virt-install.pod b/man/en/virt-install.pod
index 1552e41..b3ec9e6 100644
--- a/man/en/virt-install.pod
+++ b/man/en/virt-install.pod
@@ -172,11 +172,12 @@ may cause issues if migrating the guest to a host without an identical CPU.
 Human readable text description of the virtual machine. This will be stored
 in the guests XML configuration for access by other applications.

-=item --security type=TYPE[,label=LABEL]
+=item --security type=TYPE[,label=LABEL][,relabel=yes|no]

 Configure domain security driver settings. Type can be either 'static' or
 'dynamic'. 'static' configuration requires a security LABEL. Specifying
-LABEL without TYPE implies static configuration.
+LABEL without TYPE implies static configuration. To have libvirt automatically
+apply your static label, you must specify relabel=yes.

 =back

-- 
1.8.0.2
